src/client/Microsoft.Identity.Client/Cache/CacheInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.;
+
+namespace Microsoft.Identity.Client.Cache
+{
+    /// <summary>
+    /// Enum describing the reason for cache refresh and fetching access token from ESTS.
+    /// </summary>
+    public enum CacheInfo
+    {
+        /// <summary>
+        /// When the cache is not hit to make the request (interactive call, username password call, device code flow, etc.)
+        /// </summary>
+        None = 0,
+        /// <summary>
+        /// When the token request goes to ESTS because force_refresh was set to true
```

Comment:
Let's not mention ESTS here, as it's an internal acronym for AAD. I think "Identity Provider" is the better term.